container_files/tutorials/Virtual manipulation of datasets.ipynb

```diff
@@ -0,0 +1,924 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Virtual manipulation of datasets tutorial\n",
+    "\n",
+    "In this tutorial, we will be using two [Dataset types](../../../../doc/#builtin/datasets/Datasets.md.html), namely [Sampled Dataset](../../../../doc/#builtin/datasets/SampledDataset.md.html) and [Transposed Dataset](../../../../doc/#builtin/datasets/TransposedDataset.md.html). This tutorial assumes familiarity with [Procedures](../../../../doc/#builtin/procedures/Procedures.md.html).\n",
```

Comment:
à la fin, ajouter quelque chose comme: ", and so we suggest going through the Procedures and Functions Tutorial (https://docs.mldb.ai/ipy/notebooks/_tutorials/_latest/Procedures%20and%20Functions%20Tutorial.html) beforehand.